src/angular-cesium/services/cesium/cesium.service.ts

```diff
@@ -1,24 +1,29 @@
 import { Injectable, NgZone, Optional } from '@angular/core';
 import { ViewerFactory } from '../viewer-factory/viewer-factory.service';
 import { ViewerConfiguration } from '../viewer-configuration/viewer-configuration.service';
+import { AcMapComponent } from '../../components/ac-map/ac-map.component';
 
 /**
  *  Service that initialize cesium viewer and expose cesium viewer and scene.
  */
 @Injectable()
 export class CesiumService {
   private cesiumViewer: any;
+  private map: AcMapComponent;
 
-  constructor(private ngZone: NgZone, private viewerFactory: ViewerFactory, @Optional() private viewerConfiguration: ViewerConfiguration) {
-  }
+  constructor(private ngZone: NgZone, private viewerFactory: ViewerFactory, @Optional() private viewerConfiguration: ViewerConfiguration) {}
 
-  init(mapContainer: HTMLElement) {
+  init(mapContainer: HTMLElement, map: AcMapComponent) {
+    this.map = map;
     this.ngZone.runOutsideAngular(() => {
       const options = this.viewerConfiguration ? this.viewerConfiguration.viewerOptions : undefined;
```

Comment:
what about `ViewerConfiguration` how is it supports multi maps?

Reply:
added support through array